### PR TITLE
Just some instructions 

### PR DESCRIPTION
**Pull request by Nickeytheblade**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+https://github.com/adafruit/Adafruit_nRF52_nrfutil
+
+All of the Adafruit board definitions in the sketch through the library manager.
+
+The bsec library needs to be at (1.4.703).
+
+As well the Adafruit Bluefruit nrf51 library through the library manager.
+
+Specifically the Rak nrf (0.21.20) board version.
+#https://forum.rakwireless.com/t/just-updated-nrf-boards-for-rak-4631-broke-my-code/5189/2
```
Comment (slivingston) [inline README.md:18]: Why does this URL line should not start with `#`?

Comment (Nickeytheblade) [reply README.md:18]: Makes it one size bigger so it stands out more

Comment (Nickeytheblade) [reply README.md:18]: dose not half to

Comment (slivingston) [reply README.md]: OK, I see you changed it. let's go with it now and we can re-style later.

**Review by slivingston — APPROVED** [review]

**Pull request merged**